frontend/packages/console-dynamic-plugin-sdk/src/app/modal-support/useModal.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { LaunchModal, ModalContext } from './ModalProvider';
+
+type UseModalLauncher = () => LaunchModal;
+
+/**
+ * A hook to launch Modals.
```

Comment:
Can we add a very minimal code example? It would make the API docs better :smiley: